lib/systems/inspect.nix

```diff
@@ -48,6 +48,7 @@ rec {
     isRiscV64      = { cpu = { family = "riscv"; bits = 64; }; };
     isRx           = { cpu = { family = "rx"; }; };
     isSparc        = { cpu = { family = "sparc"; }; };
+    isSparc64      = { cpu = { family = "sparc"; bits = 64; }; };
```

Comment:
This is independently useful; could you please split it out in a separate PR?  I'll merge it.  I'm surprised we don't already have this.